packages/shared/src/mock-values.ts

```diff
@@ -17,6 +17,11 @@ const range9 = () => range(1, 11);
 
 const oneD = () => ndarray(range1().map((val) => val ** 2));
 
+const oneD_bigint = () =>
+  ndarray(
+    range8().map((_, i) => BigInt(Number.MAX_SAFE_INTEGER) + BigInt(i - 5)),
```

Comment:
I make the values cross the max safe integer threshold. This will hopefully be interesting when visualizing as line/heatmap after casting to float64/float32.